OneCommunityGlobal/HighestGoodNetworkApp
# Xiao F update teams page sorting part c

# Description
![CleanShot 2023-12-29 at 16 36 28](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/43768723/b7c5fb59-07c9-4084-8425-c96c751b7f05)


## Related PRS (if any):
To test this frontend PR you need to checkout the [#671](https://github.com/OneCommunityGlobal/HGNRest/pull/671) backend PR.


## Main changes explained:
- Update file `src/components/Teams/TeamTableHeader.jsx` for including new icons and getting onclick methods from child component
- Update file `src/components/Teams/Teams.jsx` for including new sorting methods and states for managing different sortings

## How to test:
1. check into current branch
2. do `npm install` and `npm run start:local` to run this PR locally
3. Clear site data/cache
4. log as owner/admin user (Right now, you need an owner account to toggle 'Active' status for a team)
5. go to Other Links-> Teams->  Click 'Team Names' to cycle through sorting methods: default(by modified date), ascending, descending
6. go to Other Links-> Teams->  Click 'Active' to cycle through sorting methods: default(by modified date), ascending(inactive teams on top), descending
7. When 'Team Names' is not in default sorting, clicking on 'Active' will reset 'Team Names' icon to default

## Screenshots or videos of changes:

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/43768723/d73825cf-6fe0-4861-83f5-98768729fdb0
